src/libopensc/pkcs15-pteid.c

```diff
@@ -106,22 +107,24 @@ static int sc_pkcs15emu_pteid_init(sc_pkcs15_card_t * p15card)
 	/* TODO: Use Portuguese descriptions? */
 	
 	/* Add X.509 Certificates */
-	for (i = 0; i < 4; i++) {
-		static const char *pteid_cert_names[4] = {
-				"AUTHENTICATION CERTIFICATE",
-				"SIGNATURE CERTIFICATE",
-				"SIGNATURE SUB CA",
-				"AUTHENTICATION SUB CA"
+	for (i = 0; i < 5; i++) {
+		static const char *pteid_cert_names[5] = {
+			"AUTHENTICATION CERTIFICATE",
+			"SIGNATURE CERTIFICATE",
+			"SIGNATURE SUB CA",
+			"AUTHENTICATION SUB CA",
+			"ROOT CA"
 		};
```

Comment:
I'd leave the indentation unchanged so as not to obscure the "meat" of your patch.